constants/battle_anim_constants.asm

```diff
@@ -306,191 +306,191 @@ DEF NUM_BATTLEANIMFUNCS EQU const_value
 
 ; BattleAnimFrameData indexes (see data/battle_anims/framesets.asm)
 	const_def
-	const BATTLEANIMFRAMESET_00
-	const BATTLEANIMFRAMESET_01
-	const BATTLEANIMFRAMESET_02
-	const BATTLEANIMFRAMESET_03
-	const BATTLEANIMFRAMESET_04
-	const BATTLEANIMFRAMESET_05
-	const BATTLEANIMFRAMESET_06
-	const BATTLEANIMFRAMESET_07
-	const BATTLEANIMFRAMESET_08
-	const BATTLEANIMFRAMESET_09
-	const BATTLEANIMFRAMESET_0A
-	const BATTLEANIMFRAMESET_0B
-	const BATTLEANIMFRAMESET_0C
-	const BATTLEANIMFRAMESET_0D
-	const BATTLEANIMFRAMESET_0E
-	const BATTLEANIMFRAMESET_0F
-	const BATTLEANIMFRAMESET_10
-	const BATTLEANIMFRAMESET_11
-	const BATTLEANIMFRAMESET_12
-	const BATTLEANIMFRAMESET_13
-	const BATTLEANIMFRAMESET_14
-	const BATTLEANIMFRAMESET_15
-	const BATTLEANIMFRAMESET_16
-	const BATTLEANIMFRAMESET_17
-	const BATTLEANIMFRAMESET_18
-	const BATTLEANIMFRAMESET_19
-	const BATTLEANIMFRAMESET_1A
-	const BATTLEANIMFRAMESET_1B
-	const BATTLEANIMFRAMESET_1C
-	const BATTLEANIMFRAMESET_1D
+	const BATTLEANIMFRAMESET_HIT_BIG
+	const BATTLEANIMFRAMESET_HIT
+	const BATTLEANIMFRAMESET_HIT_SMALL
+	const BATTLEANIMFRAMESET_PUNCH
+	const BATTLEANIMFRAMESET_KICK
+	const BATTLEANIMFRAMESET_PALM
+	const BATTLEANIMFRAMESET_FANG
+	const BATTLEANIMFRAMESET_PUNCH_SHAKE
+	const BATTLEANIMFRAMESET_BALL_POOF
+	const BATTLEANIMFRAMESET_POKE_BALL_1
+	const BATTLEANIMFRAMESET_POKE_BALL_2
+	const BATTLEANIMFRAMESET_POKE_BALL_3
+	const BATTLEANIMFRAMESET_POKE_BALL_4
+	const BATTLEANIMFRAMESET_POKE_BALL_5
+	const BATTLEANIMFRAMESET_DRAGON_RAGE
+	const BATTLEANIMFRAMESET_FLAMETHROWER
+	const BATTLEANIMFRAMESET_EMBER
+	const BATTLEANIMFRAMESET_BURNED
+	const BATTLEANIMFRAMESET_BLIZZARD
+	const BATTLEANIMFRAMESET_ICE
+	const BATTLEANIMFRAMESET_ICE_BEAM
+	const BATTLEANIMFRAMESET_POWDER_SNOW
+	const BATTLEANIMFRAMESET_RAZOR_LEAF_1
+	const BATTLEANIMFRAMESET_RAZOR_LEAF_2
+	const BATTLEANIMFRAMESET_EXPLOSION
+	const BATTLEANIMFRAMESET_BIG_ROCK
+	const BATTLEANIMFRAMESET_SMALL_ROCK
+	const BATTLEANIMFRAMESET_STRENGTH
+	const BATTLEANIMFRAMESET_SKULL_CROSSBONE
+	const BATTLEANIMFRAMESET_ACID
 	const BATTLEANIMFRAMESET_1E
```

Comment:
If this were used, does it have a sensible name? Has it been useful in any custom anims?
```suggestion
	const BATTLEANIMFRAMESET_1E ; unused
```